docs/guide/ingress/annotations.md

```diff
@@ -226,7 +226,10 @@ Traffic Routing can be controlled with following annotations:
 - <a name="subnets">`alb.ingress.kubernetes.io/subnets`</a> specifies the [Availability Zone](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)s that the ALB will route traffic to. See [Load Balancer subnets](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-subnets.html) for more details.
 
     !!!note ""
-        You must specify at least two subnets in different AZs. Either subnetID or subnetName(Name tag on subnets) can be used.
+        You must specify at least two subnets in different AZs unless utilizing the outpost locale, in which case a single subnet suffices. Either subnetID or subnetName(Name tag on subnets) can be used.
+
+    !!!note ""
+        You must not mix subnets subnets from different locales: availability-zone, local-zone, wavelength-zone, outpost.
```

Comment:
ALB does not support wavelength yet right?
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#subnets-load-balancer

Reply:
> You must not mix subnets subnets 

duplicate `subnets`?

Reply:
looks like wavelength should also be supported in ALB: https://aws.amazon.com/blogs/compute/enabling-load-balancing-of-non-https-traffic-on-aws-wavelength/
thank you for pointing this out.

Reply:
I saw the error about the mixed locales for regular + local zones and for regular + outposts. I just assumed it should be the same for wavelength.